platform/ddf-theme/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "ddf-theme",
+  "version": "0.0.1",
+  "description": "DDF's Bootstrap Theme",
+  "main": "index.js",
+  "scripts": {
+    "copy-bootswatch-fonts": "node node_modules/cpr/bin/cpr node_modules/bootswatch/fonts/ src/main/resources/webapp/fonts/bootstrap -o",
```

Comment:
I think for a lot of these commands, yarn adds `node_modules/.bin` to your path, so we don't need search for them in `node_modules`.